### PR TITLE
Add README

### LINKED ISSUE #15: Add README

### PR DESCRIPTION
**Pull request by forbiddenpersimmon**
# Description
Initial README for VennDiagram includes description, installation, resources, getting help, and citation information.

### Closes #15 

# Checklist

- [X] This PR does NOT contain [PHI](https://ohrpp.research.ucla.edu/hipaa/) or germline genetic data.  A repo may need to be deleted if such data is uploaded. Disclosing PHI is a [major problem](https://healthitsecurity.com/news/ucla-health-reaches-7.5m-settlement-over-2015-breach-of-4.5m).
- [X] This PR does NOT contain molecular files, compressed files, output files such as images (*e.g.* `.png`, .`jpeg`), `.pdf`, `.RData`, `.xlsx`, `.doc`, `.ppt`, or other non-plain-text files.  To automatically exclude such files using a [.gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) file, see [here](https://github.com/uclahs-cds/template-base/blob/main/.gitignore) for example.
- [X] I have read the [code review guidelines](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Guidelines) and the [code review best practice on GitHub check-list](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List).
- [X] I have set up or verified the `main` branch protection rule following the [github standards](https://confluence.mednet.ucla.edu/pages/viewpage.action?spaceKey=BOUTROSLAB&title=GitHub+Standards#GitHubStandards-Branchprotectionrule) before opening this pull request.
- [X] The name of the branch is meaningful and well formatted following the [standards](https://confluence.mednet.ucla.edu/display/BOUTROSLAB/Code+Review+Best+Practice+on+GitHub+-+Check+List), using [AD_username (or 5 letters of AD if AD is too long)]-[brief_description_of_branch].
- [ ] I have added the major changes included in this pull request to the `CHANGELOG.md` under the next release version or unreleased, and updated the date.

<!--

### CODE REVIEW

**Review by dan-knight — CHANGES_REQUESTED** [review]
Looks amazing! My only suggestion is to consider formatting the descriptive text for the example plots as bulleted lists or something similar. I think we can be even clearer than the paragraph form is currently.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+**Selected Venn diagram special cases and Euler diagrams drawn by the VennDiagram package.** Row 1, column 1: automatically drawn, customizable lines that optimize display of partial areas when individual partial areas become too small in two-set Venn diagrams. Row 1, column 2: a two-set Euler diagram showing total inclusion of one of the sets. Row 1, column 3: a two-set Euler diagram showing two distinct sets. Row 2, column 1: a three-set Euler diagram where one set has no discrete elements. Row 2, column 2: a three-set Euler diagram where one set has no discrete elements is totally included in one of the other two sets. Row 2, column 3: a three-set Euler diagram where two sets have no discrete elements and are included in a larger third set. Row 3, column 1: a three-set Euler diagram showing total inclusion of two sets that are distinct from the third set. Row 3, column 2: a three-set Euler diagram where one set is totally included in another set, which is itself totally included in the third set. Row 3, column 2: a three-set Euler diagram showing three distinct sets.
```
Comment (stefaneng) [inline README.md:23]:
```suggestion
Selected Venn diagram special cases and Euler diagrams drawn by the VennDiagram package.

  - **Row 1, column 1**: automatically drawn, customizable lines that optimize display of partial areas when individual partial areas become too small in two-set Venn diagrams.
  - **Row 1, column 2**: a two-set Euler diagram showing total inclusion of one of the sets.
  - **Row 1, column 3**: a two-set Euler diagram showing two distinct sets.
  - **Row 2, column 1**: a three-set Euler diagram where one set has no discrete elements.
  - **Row 2, column 2**: a three-set Euler diagram where one set has no discrete elements is totally included in one of the other two sets.
  - **Row 2, column 3**: a three-set Euler diagram where two sets have no discrete elements and are included in a larger third set.
  - **Row 3, column 1**: a three-set Euler diagram showing total inclusion of two sets that are distinct from the third set. 
  - **Row 3, column 2**: a three-set Euler diagram where one set is totally included in another set, which is itself totally included in the third set. 
  - **Row 3, column 2**: a three-set Euler diagram showing three distinct sets.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+![](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-35/MediaObjects/12859_2010_Article_5075_Fig3_HTML.jpg?as=webp)
+
+**A side-by-side comparison of an Euler diagram and a Venn diagram for the same hypothetical sets.** A) The Euler diagram shows only non-zero areas and can therefore be more graphically accurate. B) The Venn diagram shows the non-existent area as an area with zero content. Though this is not graphically accurate, it preserves the recognisability of a Venn diagram.
```
Comment (stefaneng) [inline README.md:27]:
```suggestion
A side-by-side comparison of an Euler diagram and a Venn diagram for the same hypothetical sets.

<!-- to get A, B -->
<ol type="a">
    <li>The Euler diagram shows only non-zero areas and can therefore be more graphically accurate.</li>
    <li>The Venn diagram shows the non-existent area as an area with zero content. Though this is not graphically accurate, it preserves the recognisability of a Venn diagram.</li>
</ol>
```

**Review by stefaneng — CHANGES_REQUESTED** [review]
Nice work! I added Dan's list ideas as suggestions.

Maybe added one section with a simple example?

``` r
library(VennDiagram)
vd <- VennDiagram::venn.diagram(list(A = 1:150, B = 121:170), filename = NULL)
grid::grid.draw(vd)
```

File: README.md [hunk 3]
```diff
@@ -14,26 +14,61 @@ VennDiagram is a R package for generating high-resolution, customizable Venn dia
...
 install.packages("VennDiagram")
 ```
+
 Or to install the latest development version from Github:
-```
+
+```R
 # install.packages("devtools")
 
 devtools::install_github("uclahs-cds/public-R-VennDiagram")
```
Comment (stefaneng) [inline README.md:74]: Move this up above the example. In the TOC it is listed as 2nd which makes sense.

**Review by stefaneng — CHANGES_REQUESTED** [review]
Just one minor change but other than that looks good!

File: README.md (hunk 3 above)
Comment (forbiddenpersimmon) [reply README.md]: Good catch, moved it back up!

**Review by stefaneng — APPROVED** [review]
LGTM but will let @dan-knight and @tyamaguchi-ucla make any final comments.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+For guidance or support with VennDiagram check out [Discussions](https://github.com/uclahs-cds/public-R-VennDiagram/discussions)
+
+See [Issues](https://github.com/uclahs-cds/public-R-VennDiagram/issues) to submit bugs, suggest new features or view current works
+
+[Pull requests](https://github.com/uclahs-cds/public-R-VennDiagram/pulls) are also open for discussion.
+
+## Citation information
+
+Chen, H., Boutros, P.C. VennDiagram: a package for the generation of highly-customizable Venn and Euler diagrams in R. BMC Bioinformatics 12, 35 (2011). https://doi.org/10.1186/1471-2105-12-35
```
Comment (stefaneng) [inline README.md:92]:
```suggestion
To cite package `VennDiagram` in publications use:

  Chen, H., Boutros, P.C. VennDiagram: a package for the generation of highly-customizable Venn and
  Euler diagrams in R. BMC Bioinformatics 12, 35 (2011). https://doi.org/10.1186/1471-2105-12-35

A BibTeX entry for LaTeX users is

```
@Article{,
    title = {VennDiagram: a package for the generation of highly-customizable Venn and Euler diagrams in R},
    journal = {BMC Bioinformatics},
    doi = {10.1186/1471-2105-12-35},
    url = {https://doi.org/10.1186/1471-2105-12-35},
    volume = {12},
    number = {35},
    year = {2011},
    month = {January},
    day = {26},
    issn = {1471-2105},
    author = {Hanbo Chen and Paul C Boutros},
  }
```
```

**Review by tyamaguchi-ucla — APPROVED** [review]
Looks good to me!

**Comment by tyamaguchi-ucla** [comment]
Anything else to add @dan-knight ?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+## Example plots
+
+How to make a simple Venn diagram:
+
+```R
+library(VennDiagram)
+vd <- VennDiagram::venn.diagram(list(A = 1:150, B = 121:170), filename = NULL)
+grid::grid.draw(vd)
```
Comment (pboutros) [inline README.md:38]: please semi-colon terminate

**Review by pboutros — APPROVED** [review]
One minor thing, LGTM!

**Comment by tyamaguchi-ucla** [comment]
@dan-knight this PR requires your approval for merging as you requested changes.

**Review by dan-knight — APPROVED** [review]

**Comment by dan-knight** [comment]
Oops, approved. Sorry for blocking!

**Pull request merged**